### PR TITLE
Use a cron/script to auth with ECR. Use the versioned image

### PR DESCRIPTION
**Pull request by joffotron**

### CODE REVIEW

File: ansible/build-node.yml [hunk 1]
```diff
@@ -2,29 +2,31 @@
...
+        minute: "*/10"
 
     - name: Pull supporting Docker images
       become: yes
-      docker_image:
-       name: "{{ item }}"
+      become_user: buildkite-agent
+      command: docker pull "{{ item }}"
       with_items:
+        - "984601232468.dkr.ecr.us-west-2.amazonaws.com/caremonkey/cm-builder:3"
```
Comment (Andythurlow) [inline ansible/build-node.yml:29]: 3? Was this meant to be version 2 to match up with the PR in the app?

Ignore this otherwise :)

**Comment by Andythurlow** [comment]
+1 pending the above.

File: ansible/build-node.yml (hunk 1 above)
Comment (joffotron) [inline ansible/build-node.yml:29]: Meant to be 3. The diff between this and v2 is small, and v2 will get pulled automatically. This way we bake the AMI machine image with 3 already pre-loaded for later on ;)

Comment (Andythurlow) [inline ansible/build-node.yml:29]: nice!

**Pull request merged**